src/app/citizenScience/textLabels/labelCheck.tpl.html

```diff
@@ -2,7 +2,7 @@
     <span>
         <a ng-click="toggleLabel()" class="main-button">
         <i class="fa"
-           ng-class="{true: 'fa-check-square-o', false : 'fa-square-o'}[$ctrl.checked.value]"
+           ng-class="{true: 'fa-check-square-o', false : 'fa-square-o'}[$ctrl.isChecked()]"
```

Comment:
this is a pretty funky way to use ng-class. Wouldn't the more idiomatic  `class="fa-square-o" ng-class="{'fa-check-square-o': $ctrl.isChecked()}"` also work?

Reply:
Above suggestion doesn't seem to work because it applies both the checked and not checked style at the same time, so I used `class="fa"
           ng-class="$ctrl.isChecked() ? 'fa-check-square-o' : 'fa-square-o'"`